perf/datarepo/create-secret-manager-secret.yaml

```diff
@@ -19,6 +19,12 @@ secrets:
       - kubeSecretKey: applicationsecret
         path: secret/dsde/datarepo/perf/helm-azure
         vaultKey: applicationsecret
+      - kubeSecretKey: synapse-us-east-sql-admin-user
+        path: secret/dsde/datarepo/dev/helm-azure
```

Comment:
Should perf have its own set of synapse credentials?

Reply:
Yes!! Thank you so much for catching that! Should be fixed now.